### ISSUE TITLE
GPT-4 Hates Backticks

### ISSUE THREAD

**Issue by Oliver-Turp** [pieces-app/support#101]
### Software

Desktop Application

### Operating System

Windows

### Your Pieces OS Version

8.0.0

### Kindly describe the bug and include as much detail as possible on what you were doing so we can reproduce the bug.

I often send code to copilot chat for syntax checking. I have a preference for using backticks over quotation marks as if I want to add additional code in later, I don't have to remove the quotes and add in the backticks
```
console.log(`this is an example`)
console.log(`this is a ${phrase}`}
```
When I do this, GPT-4 will suggest changing them but will send back identical code. This is an example I just got.
```
Yes, there are syntax errors in your code. The issue is with the template literals (strings enclosed in backticks ` `) in your code.
... 
```
In short, there's nothing wrong with using backticks, but GPT-4 really hates them.

**Comment by Oliver-Turp** [comment]
Update from today, with another example
```
Looking at your code, it seems like the issue might be with the "description" field in your "embeds" object. You're trying to use a template literal (`<@${discordId}>`), but it's not inside backticks (`), which is causing a syntax error.
```
GPT-4 returned the code I'd used, which was in backticks, and expressed that using backticks would fix my issue. It then returned the same code I'd sent in.

**Comment by jimbobbennett** [comment]
This is still happening with Pieces Desktop 3.2.2 and Pieces OS 10.1.12. Looks like it is down to how we process the backticks in the copilot input, I assume we're converting from markdown.
